docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -39,6 +39,79 @@ The Fairlearn package is most applicable to two kinds of harms:
+
+- **Unobservable theoretical construct** – An idea or concept that is unobservable and cannot be directly measured but must instead be inferred through observable measurements defined in a measurement model. 
+
+- **Measurement model** – The method and approach used to measure the unobservable theoretical construct.
+
+- **Construct reliability** – This is closely related to construct validity (detailed below) and can be thought of as the extent to which the measurements of an unobservable theoretical construct remain the same when measured at different points in time. A lack of construct reliability can either be due to a misalignment between the understanding of the unobservable theoretical construct and the methods being used to measure that construct, or to changes to the construct itself.
+
+- **Construct validity** – This can be thought of as the extent to which the measurement model can be used to produce meaningful and useful results.
+
+Key Term Examples  - Unobservable theoretical constructs and Measurement models
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- **Fairness** is an example of an unobservable theoretical construct. Several measurement models exist for measuring fairness, including demographic parity. See Fairlearn Metrics for more examples of measurement models for measuring fairness. 
+
+- **Teacher effectiveness** is an example of an unobservable theoretical construct. Common measurement models include student performance on standardized exams and qualitative feedback for the teacher’s students.
+
+- **Socioeconomic status** is an example of an unobservable theoretical construct. A common measurement model includes annual household income. 
+
+- **Patient benefit** is an example of an unobservable theoretical construct. A common measurement model involves patient care costs.
+
+**Note:** We cite several examples of unobservable theoretical constructs and measurement models for the purpose of explaining the key terms outlined above.  Please reference Measurement and Fairness for more detailed examples.
+ 
+What is Construct Validity? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Though the Measurement and Fairness paper explores both Construct reliability and Construct validity, we focus our 
+exploration below on Construct Validity. We note that both play an important role in understanding fairness in sociotechnical 
+contexts. With that said, Measurement and Fairness offers a fairness-oriented conceptualization of construct validity, that 
+is helpful in thinking about fairness in sociotechnical contexts. We capture the idea in seven key parts that when combined 
+can serve as a framework for analyzing an AI task and attempting to establish construct validity:
+
+1. **Face validity** – On the surface, how plausible does the measurements produced by the measurement model look?
```

Comment:
Please add an example for each component of construct validity.

Reply:
Part of the thinking was that this glossary would be used and referenced by other contributors, who could then utilize the terms above to create examples. Curious to hear other's thoughts on if in addition to the pending educational material that will use the above in specific example, we need to add example here too? 

@mmadaio
@hannawallach  
Any thoughts?

Reply:
Ah, I had a similar comment here too - I'm sort of of two minds about it. I definitely agree that examples are going to be critical to get these concepts across. But that could also be done in-depth in future PRs. I wonder whether a short (e.g., 1-2 sentence) example using the examples mentioned earlier (e.g., teacher effectiveness, etc) might be a useful starting point? 

I could see an argument to either use fairness as the running example (since... it is Fairlearn after all :), or for _not_ using fairness as the running example, since it might be too close to home for readers, and they'll be thinking about their own context etc, and not the concept itself.